net-dns/pihole/pihole-5.3.1-r2.ebuild

```diff
@@ -11,10 +11,11 @@ SRC_URI="https://github.com/pi-hole/pi-hole/archive/v${PV}.tar.gz -> ${P}.tar.gz
 LICENSE="EUPL-1.2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="cron www"
+IUSE="cron www ftl"
```

Comment:
Can we not have the `ftl` in IUSE? It's only used to pull in a package and does not affect configuration/build.

Reply:
Two packages _net-dns/pihole_ and _net-dns/pihole-ftl_ are mutually connected.
Each of them can not be used without another.
So user, which install one of them, in 100% times install another.

If overlay admin want this - I remove this use flag.
But I think, that this use flag MUST be present and activated by default (in form of **+ftl**).

Reply:
Remove the IUSE ftl flag, and make `net-dns/pihole-ftl` a required dependency.

Reply:
Actually this change is not required. On line 20, `net-dns/${PN}-ftl` is already there in RDEPEND. So just remove this change.